charts/rancher-monitoring/v0.1.5/questions.yml

```diff
@@ -0,0 +1 @@
+rancher_min_version: 2.4.9-rc1
```

Comment:
Does this mean there is more than 1 chart available for monitoring for k8s 1.16? I'd expect that this would be a min version of 2.4.14-rc1 and there would be a Rancher max version for the v0.1.4 chart.

Reply:
Yes, at the moment a k8s 1.16+ cluster could still choose to use the v0.1.x line if they wanted to use the older version of Prometheus Operator. This is true in dev-v2.5 as well as seen here: https://github.com/rancher/system-charts/pull/421/files#r585919224

Reply:
https://github.com/rancher/system-charts/pull/424